Xamarin.Essentials/MediaPicker/MediaPicker.uwp.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+
+namespace Xamarin.Essentials
+{
+    public static partial class MediaPicker
+    {
+        static readonly string[] defaultVideoFileTypes = new[] { ".mp4", ".wmv", ".avi" };
+        static readonly string[] defaultImageFileTypes = new[] { ".jpeg", ".jpg", ".png", ".gif", ".bmp" };
+
+        static async Task<MediaFile> PlatformShowPhotoPickerAsync(MediaPickerOptions options)
+        {
+            var picker = new FileOpenPicker();
+
+            // set picker properties
+            foreach (var filter in defaultImageFileTypes)
+                picker.FileTypeFilter.Add(filter);
+            picker.SuggestedStartLocation = PickerLocationId.PicturesLibrary;
+            picker.ViewMode = PickerViewMode.Thumbnail;
+
+            // show the picker
+            var result = await picker.PickSingleFileAsync();
+
+            // cancelled
+            if (result == null)
+            {
+                OnMediaPicked(new MediaPickedEventArgs(true));
+                return null;
```

Comment:
We want to (optionally) fire the global event and then resure for the task value.